admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/entity/ValidDocument.java

```diff
@@ -43,12 +45,13 @@ public class ValidDocument extends BaseEntity implements Serializable {
 	private static final long serialVersionUID = -3111581667845281498L;
 
 	@Id
-	@AttributeOverrides({
-			@AttributeOverride(name = "docTypeCode", column = @Column(name = "doctyp_code", nullable = false, length = 36)),
-			@AttributeOverride(name = "docCategoryCode", column = @Column(name = "doccat_code", nullable = false, length = 36)) })
-
+    @Column(name = "doctyp_code", nullable = false, length = 36)
+	@DocTypeCode(message = "docType is Invalid")

```

Comment:
It is always better to set error_code in the message instead of message directly.